lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -224,16 +228,27 @@ class Simulator {
 
     if (node.type !== Node.TYPES.NETWORK) throw new Error('Unsupported');
 
-    const record = /** @type {NetworkNode} */ (node).record;
+    const networkNode = /** @type {NetworkNode} */ (node);
     const timingData = this._nodeTimings.get(node);
-    // If we're estimating time remaining, we already acquired a connection for this record, definitely non-null
-    const connection = /** @type {TcpConnection} */ (this._acquireConnection(record));
-    const calculation = connection.simulateDownloadUntil(
-      record.transferSize - timingData.bytesDownloaded,
-      {timeAlreadyElapsed: timingData.timeElapsed, maximumTimeToElapse: Infinity}
-    );
 
-    const estimatedTimeElapsed = calculation.timeElapsed + timingData.timeElapsedOvershoot;
+    let timeElapsed = 0;
+    if (networkNode.fromDiskCache) {
```

Comment:
Given the new logic I think we should spilt `_estimateTimeRemaining` into a few pieces -- it's gettin a little long.

Reply:
aye, prefer that happen now? wanted to keep the diff a little simpler though I suppose with separate commits that's still the case

Reply:
ya let's do it now. i can follow these changes pretty ez.